src/main/java/core/basesyntax/strategy/impl/BalanceStrategy.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.db.StorageFruits;
+import core.basesyntax.strategy.Strategy;
+
+public class BalanceStrategy implements Strategy {
+    @Override
+    public void makeOperation(String fruit, int value) {
+        StorageFruits.fruits.put(fruit, value);
```

Comment:
hmmmm, i think this is not right, that from service we are put everything straight in Storage, better do it threw Dao, but okay, maybe better mentor later will say if its okay or not^_^